unfoldingWord/translationCore
# Added Persistence To Groups Data

#### This pull request addresses:
#1011 

#### How to test this pull request:
To test this PR:
checkout js_groups in tW
start with a fresh project
load tW (observer wait time)
reload and see if it is faster


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1063)
